.github/workflows/cicd.yaml

```diff
@@ -249,6 +249,11 @@ jobs:
         run: |
           bash tests/integration/integration-test.sh "${{ matrix.integration-test-arg }}"
         shell: bash
+      - name: Display k8s logs if integration test failed
+        if: failure()
+        run: |
+          kubectl logs -n connaisseur -lapp.kubernetes.io/name=connaisseur --prefix=true
```

Comment:
what does `--prefix` do?

Reply:
In the example link, it generates logs such as
```[pod/connaisseur-deployment-7f5ddc58c5-vh8c6/connaisseur] [2022-02-18 09:50:25,270] INFO: {'message': 'automatic child approval for "docker.io/library/redis@sha256:5b98e32b58cdbf9f6b6f77072c4915d5ebec43912114031f37fa5fa25b032489".', 'context': {'user': 'system:serviceaccount:kube-system:replicaset-controller', 'operation': 'CREATE', 'kind': 'Pod', 'name': 'redis-with-many-instances-7dd4c94796-', 'namespace': 'default', 'image': 'docker.io/library/redis@sha256:5b98e32b58cdbf9f6b6f77072c4915d5ebec43912114031f37fa5fa25b032489'}}```
`--prefix` adds the `[pod/connaisseur-deployment-7f5ddc58c5-vh8c6/connaisseur] ` part

Reply:
would also benefit from #541